src/main/java/org/example/homework4/MinutesCalculator.java

```diff
@@ -0,0 +1,11 @@
+package org.example.homework4;
+
+public class MinutesCalculator {
+    public void printYearsAndDays(long minutes) {
+        if (minutes > 0) {
+            long years = minutes / (365 * 1440);
```

Comment:
without a context 365 and 1440 are magic numbers. What is 1440? They could be extracted as a constants or use more variables to improve readability. e.g.
```
            long hours = minutes / 60;
            long days = hours / 24;
            long years = days / 365;
            long remainingDays = days % 365;
```